lib/metanorma/plugin/lutaml/liquid_templates/_packages_data_dictionary.liquid

```diff
@@ -0,0 +1,123 @@
+{% for package in context.packages %}
+{% assign package_name = package.name | downcase | replace: ":", "" | replace: " ", "_" %}
+{% capture equalsigns %}{% for count in (1..depth) %}={% endfor %}{% endcapture %}{{equalsigns}} {{ package.name }}
+{{equalsigns}}= {{ package.name }}
+
+[cols="1a,4a"]
+|===
+
+h|Description:    | {{ package.definition }}
+h|Parent Package: | {{ context.name }}
+h|Stereotype:     | «{{ package.stereotype }}»
+
+|===
+
+{% assign feature_types = package.classes | where: "stereotype", "FeatureType" %}
+{% assign object_types = package.classes | where: "stereotype", "ObjectType" %}
+{% assign basic_types = package.classes | where: "stereotype", "BasicType" %}
+{% assign unions = package.classes | where: "stereotype", "Union" %}
+{% assign code_lists = package.classes | where: "stereotype", "CodeList" %}
+{% if feature_types.size > 0 or object_types.size > 0 %}
+{{equalsigns}}= Classes
+
+{% include "packages_classes", classes: feature_types %}
+{% include "packages_classes", classes: object_types %}
+{% endif %}
+
+{% if basic_types.size > 0 %}
+{{equalsigns}}= Basic Types
+
+{% include "packages_classes", classes: basic_types %}
+{% endif %}
+
+{% if unions.size > 0 %}
+{{equalsigns}}= Unions
+
+{% include "packages_classes", classes: unions %}
+{% endif %}
+
+{% if code_lists.size > 0 %}
+{{equalsigns}}= Code Lists
+
+{% include "packages_classes", classes: code_lists %}
+{% endif %}
+
+{% if package.data_types.size > 0 %}
+{{equalsigns}}= Data Types
+
+{% for data_type in package.data_types %}
+{% capture generalizations %}{% for assoc in data_type.associations | where: "member_end_type", "generalization" %}{{ assoc.member_end }} {% endfor %}{% endcapture %}
+[[{{ data_type.name }}-section]]
+[cols="1a"]
+|===
+|*{{ data_type.name }}*
+|[cols="1,4",frame=none,grid=none]
+!===
+!{nbsp}{nbsp}{nbsp}{nbsp}Definition: ! {{ data_type.definition }}
+!{nbsp}{nbsp}{nbsp}{nbsp}Subclass of: ! {{ generalizations }}
+!{nbsp}{nbsp}{nbsp}{nbsp}Stereotype: !  «DataType»
+!===
+{% if data_type.associations.size > 0 %}
+{% capture rendered_associations %}
+{% for assoc in data_type.associations %}
+{% if assoc.member_end_attribute_name.size > 0 %}
+{% capture cardinality %}{% if assoc.member_end_cardinality.min == 'C' %}[0..{{ assoc.member_end_cardinality.max }}]{% elsif assoc.member_end_cardinality.min == 'M' and assoc.member_end_cardinality.max == '1' %}[1..1]{% else %}[{{ assoc.member_end_cardinality.max }}]{% endif %}{% endcapture %}
+!{set:cellbgcolor:#FFFFFF} {{ assoc.member_end_attribute_name }}  !<<{{assoc.member_end}}-section,{{assoc.member_end}}>> {{ cardinality }} ! {{ assoc.definition }}
```

Comment:
We need to remove all non-Metanorma styles like `!{set:cellbgcolor:#FFFFFF}`

Reply:
Done, removed